### ISSUE TITLE
column offset missing from migration docs

### ISSUE THREAD

**Issue by vanillajonathan** [twbs/bootstrap#32263]
In Bootstrap 3 there used to be column offsets classes such as `col-sm-offset-2`, in Bootstrap 4 these were changed to `offset-sm-2`, however the migration document fails to mention this.

https://getbootstrap.com/docs/3.4/css/#grid-responsive-resets
https://getbootstrap.com/docs/4.5/layout/grid/#offsetting-columns

**Comment by ffoodd** [comment]
This is mentionned in the migration guide: https://getbootstrap.com/docs/4.5/migration/#grid-system-1

Components documentation is the v4 one, migrating to v4 has its own page.

**Comment by vanillajonathan** [comment]
@ffoodd There is no mention of the word "offset" anywhere in that migration document.

**Comment by ffoodd** [comment]
Indeed, since it applied to all `.col` modifiers — and `.col` is the entry point here:

> Renamed the responsive grid modifier classes from `.col-{breakpoint}-{modifier}-{size}` to `.{modifier}-{breakpoint}-{size}` for simpler grid classes.

Could have been more detailed for sure, however we're already working on the migration guide from v4 to v5 so won't change this one—but will try to be more exhaustive with the new guide.

BTW it also duplicates# 22052.

**Comment by vanillajonathan** [comment]
Oh, thanks, I missed that while searching using <kbd>Ctrl</kbd>+<kbd>F</kbd> for the string "offset".
